tdrs-backend/tdpservice/middleware.py

```diff
@@ -13,3 +15,15 @@ def __call__(self, request):
         response = self.get_response(request)
         add_never_cache_headers(response)
         return response
+
+
+class SessionMiddleware(SessionMiddleware):
+    def process_response(self, request, response):
+        response = super(SessionMiddleware, self).process_response(request, response)
+
+        # if settings.SESSION_COOKIE_NAME in response.cookies:
+        #     response.cookies[settings.SESSION_COOKIE_NAME]['samesite'] = 'None'
```

Comment:
I should try to run this, it might be related to the contexts I was aware of the app not working in.